python-regression/tests/features/machine1/1_api_tests.feature

```diff
@@ -250,3 +250,25 @@ Feature: Test API calls on Machine 1
 
 		Then a response for "findTransactions" should exist
 
+	Scenario: CheckConsistency on transaction that is below max depth
+		Sends one transaction followed by multiple milestones such that
+		the transaction becomes below max depth. Then calls checkConsistency
+
+		Given a transaction is generated and attached on "nodeA-m1" with:
+			|keys       |values				|type           |
+			|address    |TEST_STORE_ADDRESS	|staticValue    |
```

Comment:
Gal generally likes for all these columns to line up with one another all the way through the file. Makes it easier to read and understand which attributes are under which columns.

Reply:
Btw, can you give tips about lining them up?
I remember it is complicated

Reply:
The main thing really is just to make sure your tab settings are set to tab and not space and make sure to line up everything with tab.